docs/package.json

```diff
@@ -19,85 +19,88 @@
-    "@pluralsight/ps-design-system-tag": "^9.0.2",
-    "@pluralsight/ps-design-system-tagsinput": "^1.1.3",
-    "@pluralsight/ps-design-system-text": "^18.1.2",
-    "@pluralsight/ps-design-system-textarea": "^8.1.3",
-    "@pluralsight/ps-design-system-textinput": "^10.1.3",
-    "@pluralsight/ps-design-system-theme": "^7.1.2",
-    "@pluralsight/ps-design-system-tooltip": "^7.1.2",
-    "@pluralsight/ps-design-system-typeahead": "^7.1.3",
-    "@pluralsight/ps-design-system-util": "^7.1.2",
-    "@pluralsight/ps-design-system-verticaltabs": "^6.0.2",
-    "@pluralsight/ps-design-system-viewtoggle": "^7.1.3",
+    "@pluralsight/ps-design-system-note": "^8.1.4",
+    "@pluralsight/ps-design-system-position": "^6.1.4",
+    "@pluralsight/ps-design-system-radio": "^8.1.4",
+    "@pluralsight/ps-design-system-row": "^13.0.3",
+    "@pluralsight/ps-design-system-screenreaderonly": "^3.1.4",
+    "@pluralsight/ps-design-system-scrollable": "^4.1.3",
+    "@pluralsight/ps-design-system-searchinput": "^8.0.4",
+    "@pluralsight/ps-design-system-starrating": "^8.0.3",
+    "@pluralsight/ps-design-system-steps": "^2.1.4",
+    "@pluralsight/ps-design-system-switch": "^7.1.3",
+    "@pluralsight/ps-design-system-tab": "^17.1.4",
+    "@pluralsight/ps-design-system-table": "^13.1.0",
+    "@pluralsight/ps-design-system-tag": "^9.0.3",
+    "@pluralsight/ps-design-system-tagsinput": "^1.2.0",
+    "@pluralsight/ps-design-system-text": "^18.1.3",
+    "@pluralsight/ps-design-system-textarea": "^8.1.4",
+    "@pluralsight/ps-design-system-textinput": "^10.1.4",
+    "@pluralsight/ps-design-system-theme": "^7.1.3",
+    "@pluralsight/ps-design-system-tooltip": "^7.1.3",
+    "@pluralsight/ps-design-system-typeahead": "^7.1.4",
+    "@pluralsight/ps-design-system-util": "^7.2.0",
+    "@pluralsight/ps-design-system-verticaltabs": "^6.0.3",
+    "@pluralsight/ps-design-system-viewtoggle": "^7.1.4",
     "body-parser": "^1.19.0",
-    "bunyan": "^1.8.14",
+    "bunyan": "^1.8.15",
     "dotenv": "^8.2.0",
     "downshift": "^6.1.0",
     "express": "^4.17.1",
     "gatsby-plugin-express": "^1.1.6",
-    "gatsby-plugin-glamor": "^2.6.0",
-    "gatsby-plugin-react-helmet": "^3.6.0",
-    "gatsby-plugin-sitemap": "^2.8.0",
-    "gatsby-plugin-styled-jsx": "^3.6.0",
-    "gatsby-remark-autolink-headers": "^2.7.0",
-    "gatsby-remark-table-of-contents": "^0.2.1",
-    "gatsby-transformer-remark": "^2.12.0",
+    "gatsby-plugin-glamor": "^2.10.0",
+    "gatsby-plugin-react-helmet": "^3.10.0",
+    "gatsby-plugin-sitemap": "^2.12.0",
+    "gatsby-plugin-styled-jsx": "^3.10.0",
+    "gatsby-remark-autolink-headers": "^2.11.0",
+    "gatsby-remark-table-of-contents": "^1.0.0",
+    "gatsby-transformer-remark": "^2.16.1",
     "glamor": "^2.20.40",
-    "helmet": "^4.2.0",
+    "helmet": "^4.4.1",
     "http-errors": "^1.8.0",
-    "iframe-resizer-react": "^1.0.4",
-    "newrelic": "^6.13.0",
+    "iframe-resizer-react": "^1.1.0",
+    "newrelic": "^7.1.2",
```

Comment:
Have you accounted for the breaking changes here?

Reply:
nope Let me roll that one back since it's an external service.